### ISSUE TITLE
font-size changing

### ISSUE THREAD

**Issue by luca-saggese** [soaple/stickyboard#3]
Hi,
i've adapted Covid-19 dashboard built with your dashboard with italian data, however while changing the region some NumberWidgets enlarge the font automatically, and i'm not able to understand why.
you can have a look at the dashboard here: http://www.thingsmind.com:3001/statistics/italydashboard

changing the "Regione" the widget with "Guariti" and "Deceduti" does enlarge the value font at least in my mac running safari 13.0.1

thanks for sharing your work

**Comment by soaple** [comment]
Hello,
I visited your dashboard, and it's very amazing work 😀

Anyway, you have to understand the core concept of StickyBoard.
(https://docs.stickyboard.co.kr/abstract/core-concepts)

In StickyBoard, every component is responsive.
I think you already know about it.
So, you can change the size and location of the component by clicking the edit button on the right bottom menu.

Equally, NumberWidgets is also responsive.
It uses [**react-textfit**](https://github.com/malte-wessel/react-textfit) package for resizing text automatically.
In conclusion, the size of the text in NumberWidgets can change depends on text length or sticker size.

I hope my response can help you.
Thank you!
